packages/core/src/stack-layout/StackLayout.tsx

```diff
@@ -27,11 +46,42 @@ type StackLayoutComponent = <T extends ElementType = "div">(
 
 export const StackLayout: StackLayoutComponent = forwardRef(
   <T extends ElementType = "div">(
-    { children, ...rest }: StackLayoutProps<T>,
+    {
+      children,
+      className,
+      direction = "column",
+      gap,
```

Comment:
changing 'gap' doesn't seem to be affecting the gap between items in the Stack. Does gap need to be passed to FlexLayout?